docs/content/workers.md

```diff
@@ -4,7 +4,7 @@
 -----
 
 
-## Starting workers
+## Workers
```

Comment:
changed that because that section is a generic description of a worker and not about how to start one